website/content/en/preview/getting-started/getting-started-with-terraform/_index.md

```diff
@@ -151,7 +151,7 @@ module "eks" {
 
       iam_role_additional_policies = [
         # Required by Karpenter
-        "arn:${local.partition}:iam::aws:policy/AmazonSSMManagedInstanceCore"
```

Comment:
Changing it to `aws` will break anyone using these instructions in non-aws partitions (cn-north-1 / cn-northwest-1). 

Not super familiar with terraform but it sounds like `${local.partition}` didn't get templatized properly. I'll try and look

Reply:
I wonder if `data.aws_partition.current.partition` is the right thing to be using [as per these docs](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/partition) and based on some [of my searching](https://github.com/search?q=org%3Aterraform-aws-modules+aws_partition&type=code).

@bryantbiggs does this make sense?